pkg/istio-agent/xds_proxy.go

```diff
@@ -142,6 +142,8 @@ func initXdsProxy(ia *Agent) (*XdsProxy, error) {
 			LocalHostAddr: localHostAddr,
 		}
 	}
+
+	cache := wasm.NewLocalFileCache(constants.IstioDataDir, wasm.DefaultWasmModulePurgeInterval, wasm.DefaultWasmModuleExpiry, ia.cfg.WASMInsecureRegistries)
```

Comment:
nit:   wasm.DefaultWasmModulePurgeInterval, wasm.DefaultWasmModuleExpiry no need to pass